.builds/alpine.yml

```diff
@@ -38,3 +38,9 @@ tasks:
       cd sway
       meson configure build -Dxwayland=disabled
       ninja -C build
+  - build-static: |
+      cd sway
+      mkdir subprojects
+      ln -s ../../wlroots subprojects/wlroots
+      meson configure build --default-library=static
```

Comment:
I think we need to force Meson to use the subproject, or else it'll pick up the one installed system-wide.

Reply:
It appears that `meson configure` was unable to change dependency source, even with force flags. Now we wipe the build dir and set it up again, with a force-fallback for good measure.

Reply:
Was `meson configure build/ --cleancache` not enough?

Reply:
It doesn't seem like it's enough.

Reply:
Same conclusion in my attempts (https://builds.sr.ht/~kennylevinsen/job/531097#bottom). :/